tests/monotouch-test/MediaAccessibility/ImageCaptioningTest.cs

```diff
@@ -32,7 +32,12 @@ public void GetCaption ()
 			using (NSUrl url = new NSUrl (NetworkResources.MicrosoftUrl)) {
 				var s = MAImageCaptioning.GetCaption (url, out var e);
 				Assert.Null (s, "remote / return value");
-				Assert.Null (e, "remote / no error"); // weird should be an "image on disk"
+				if (e != null && e.Description.Contains ("Invalid url:")) {
+					Assert.Null (e, "remote / no internet connection"); // could not connect to the network, fail and add a nice reason
```

Comment:
```suggestion
					Assert.Null (e, "remote / invalid url"); // could not connect to the network, fail and add a nice reason
```

Reply:
Since `e != null` no need to `Assert.Null (e, ...` just `Assert.Fail`
For the message maybe `"Ignore this failure when network is down"` just to be very clear that it's an expected failure only in this specific case